CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## master / unreleased
 
 * [FEATURE] Ruler: Add `external_labels` option to tag all alerts with a given set of labels.
+* [BUGFIX] Fix incorrectly named `cortex_cache_fetched_keys` and `cortex_cache_hits` metrics. #4686
```

Comment:
I suggest to both mention the old and new name, to help users better understand the change.